### DISCUSSION TITLE
Ability to retain at least one backup with schedule?

### DISCUSSION THREAD

**Discussion by jmreicha** [vmware-tanzu/velero#7217] [Community support Q&A] [answered]
Is it possible to configure scheduled backups not to expire if, for example, backups have been failing and there is only one valid backup? Looking at docs it seems like the TTL will expire a backup no matter what, but I’d like to ensure there is always at least one available, even if the backups start failing at some point.

Is there an option for this?

**Comment by draghuram** [comment, marked as answer]
Not currently. Backups are deleted when TTL expires.

**Reply by jmreicha** [reply]
That's a bummer but understandable, thanks for letting me know!
